### PR TITLE
fix arm64 buildbreak after oopjit

### PR DESCRIPTION
**Pull request by MikeHolman**

### CODE REVIEW

File: lib/Common/Memory/Chakra.Common.Memory.vcxproj [hunk 1]
```diff
@@ -45,6 +45,11 @@
       <!-- Since there are more then one XDataAllocator.cpp, we need to set them output into different directory, even when they are ExcludedFromBuild -->
       <ObjectFileName Condition="'$(Platform)'!='ARM'">$(IntDir)\arm</ObjectFileName>
     </ClCompile>
+    <ClCompile Include="$(MSBuildThisFileDirectory)arm64\XDataAllocator.cpp">
```
Comment (tcare) [inline lib/Common/Memory/Chakra.Common.Memory.vcxproj]: @curtisman does this look OK to you?

Comment (tcare) [inline lib/Common/Memory/Chakra.Common.Memory.vcxproj]: Ah this should have been for the output directory part below

**Comment by rajatd** [comment]
:clock1:

**Comment by satheeshravi** [comment]
:shipit:

File: lib/Common/Memory/arm64/XDataAllocator.cpp [hunk 2]
```diff
@@ -3,87 +3,58 @@
...
-        Assert(pdata->UnwindData != 0);
-        Assert(pdata->BeginAddress != 0);
-        pdata->BeginAddress = (DWORD)((ULONG_PTR)pdata->BeginAddress - functionStart);
-        if(pdata->Flag != 1) // if it is not packed unwind data
-        {
-            pdata->UnwindData = (DWORD)((ULONG_PTR)pdata->UnwindData - functionStart);
-        }
-    }
-    Assert(allocation.functionTable == null);
+    __debugbreak();
```
Comment (dilijev) [inline lib/Common/Memory/arm64/XDataAllocator.cpp]: Why are we replacing all the implementations with a no-op/break depending on configuration?

Comment (MikeHolman) [inline lib/Common/Memory/arm64/XDataAllocator.cpp]: the implementations were just copy/pasted from arm32, and they would not work with arm64.

File: lib/Common/Memory/Chakra.Common.Memory.vcxproj (hunk 1 above)
Comment (curtisman) [inline lib/Common/Memory/Chakra.Common.Memory.vcxproj]: Yeah... this looks ok.

**Pull request merged**